packages/labs/analyzer/src/lib/javascript/classes.ts

```diff
@@ -0,0 +1,25 @@
+/**
+ * @license
+ * Copyright 2022 Google LLC
+ * SPDX-License-Identifier: BSD-3-Clause
+ */
+
+/**
+ * @fileoverview
+ *
+ * Utilities for working with classes
+ */
+
+import ts from 'typescript';
+import {ClassDeclaration} from '../model.js';
```

Comment:
It seems a little strange that you're factoring out a factory for getting a `ClassDeclaration`, but the `ClassDeclaration` class itself is still defined in the monolithic "model.js" module. Should that be broken up as well and perhaps go here?

Reply:
I like a separation between the model and the analysis functions that produce them. We'll have other production code paths when we add custom-element-manifest into the mix soon.